mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Surgex.Mixfile do
   def project do
     [
       app: :surgex,
-      version: "2.24.1",
+      version: "2.25.0",
```

Comment:
Unfortunately, your changes are not minor. We'll have to do major release here which means you'll have to remove all deprecated modules - please read other comments.